vim-patch:8.1.2184,8.2.3804: option context is not copied when splitting a window

zeertzjq opened the pull request:
#### vim-patch:8.1.2184: option context is not copied when splitting a window

Problem:    Option context is not copied when splitting a window. (Daniel
            Hahler)
Solution:   Copy the option context, so that ":verbose set" works.
https://github.com/vim/vim/commit/cfb381421f8be7d6cb4e7dac5b827b23467d3e53

#### vim-patch:8.2.3804: script context not set when copying 'swf' and 'ts'
    
Problem:    Script context not set when copying 'swf' and 'ts'.
Solution:   Use COPY_OPT_SCTX with the right argument. (closes vim/vim#9347)
https://github.com/vim/vim/commit/6206877c511c636cbeb2a2b911451af316b62d00

seandewar commented on src/nvim/option.c:
```diff
@@ -6380,35 +6403,58 @@ void buf_copy_options(buf_T *buf, int flags)
       buf->b_p_cfu = vim_strsave(p_cfu);
+      COPY_OPT_SCTX(buf, BV_CFU);
       buf->b_p_ofu = vim_strsave(p_ofu);
+      COPY_OPT_SCTX(buf, BV_CFU);
```
```suggestion
      COPY_OPT_SCTX(buf, BV_OFU);
```

zeertzjq committed:
vim-patch:8.1.2184: option context is not copied when splitting a window

Problem:    Option context is not copied when splitting a window. (Daniel
            Hahler)
Solution:   Copy the option context, so that ":verbose set" works.
            (closes vim/vim#5066)
https://github.com/vim/vim/commit/cfb381421f8be7d6cb4e7dac5b827b23467d3e53

vim-patch:8.2.3804: script context not set when copying 'swf' and 'ts'

Problem:    Script context not set when copying 'swf' and 'ts'.
Solution:   Use COPY_OPT_SCTX with the right argument. (closes vim/vim#9347)
https://github.com/vim/vim/commit/6206877c511c636cbeb2a2b911451af316b62d00
